src/main/java/hudson/plugins/sauce_ondemand/credentials/SauceCredentials.java

```diff
@@ -138,6 +139,23 @@ public String getRestEndpoint() {
         return this.restEndpoint;
     }
 
+    @NonNull
+    public String getRestEndpointName() {
+        if (this.restEndpoint == null) {
+            return "US";
+        }
+
+        switch (this.restEndpoint) {
+            case "https://eu-central-1.saucelabs.com/":
+                return "US";
```

Comment:
Hello! This introduced a regression, should be:
```suggestion
                return "EU";
```

Please let me know if you plan to open a PR @alexh-sauce, or if not I can. Thanks!

Reply:
:facepalm: 

Fixed in #103.